source/facter/3.1/release_notes.md

```diff
@@ -5,19 +5,65 @@ title: "Facter 3.1: Release Notes"
 
 This page documents the history of the Facter 3.1 series. (Previous version: [Facter 3.0 release notes](../3.0/release_notes.html).)
 
-Facter 3.1.0
------
+## Facter 3.1.1
+
+Released October 29, 2015.
+
+Shipped in puppet-agent version 1.2.7.
+
+Facter 3.1.1 fixes several bugs and introduces support for AIX 5.3, 6.1, and 7.1, and Solaris 11.
+
+* [Fixes for Facter 3.1.1]()
+* [Introduced in Facter 3.1.1]()
```

Comment:
introduced in facter 3.1.1: https://tickets.puppetlabs.com/issues/?filter=15771
fixed in facter 3.1.1: https://tickets.puppetlabs.com/issues/?filter=15777